### PR TITLE
N41: Add Digital Extinction (#1233)

### PR DESCRIPTION
**Pull request by Indy2222**
_Part of #1233_

### CODE REVIEW

File: content/news/041/index.md [hunk 1]
```diff
@@ -69,6 +69,41 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Digital Extinction] ([GitHub][de-github], [Discord][de-discord],
+[Reddit][de-reddit]) by [@Indy2222] is a 3D real-time strategy game made with
+[Bevy].
+
+This month's update is somewhat smaller but there has been some important
+progress in multiplayer.
+
+Here is the summary of the changes since the last update. It consists of commit
+range `28d1fce..3dbac4e` (2022-12-02–2022-12-29). There were 26 non-merge
```
Comment (ozkriff) [inline content/news/041/index.md:85]: The section LGTM overall but the "It consists of commit ... commits in total" part seems unnecessary? or do you have some reason to be specified so precisely?

Comment (Indy2222) [reply content/news/041/index.md]: Ok, removed.

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Tnx, merged!

**Pull request merged**